sycl/unittests/SYCL2020API/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 # Enable exception handling for these unit tests
 set(LLVM_REQUIRES_EH 1)
-add_sycl_unittest(SpecConstantsTests OBJECT
-  DefaultValues.cpp
+add_sycl_unittest(SYCL2020APITests OBJECT
+  SYCL2020APITests.cpp
```

Comment:
I think it would be nice to separate SYCL 2020 API tests by feature i.e. one source file per feature:
SpecConstants.cpp, BackendInterop.cpp, KernelBundle.cpp, etc.

Reply:
Created a separate PR for tests refactoring: https://github.com/intel/llvm/pull/4021